src/Easify.Excel/Extensions/FileHelper.cs

```diff
@@ -0,0 +1,39 @@
+// This software is part of the Easify.Ef Library
+// Copyright (C) 2018 Intermediate Capital Group
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// 
+
+
+using System.IO;
+
+namespace Easify.Excel.Extensions
+{
+    public static class FileHelper
+    {
+        public static MemoryStream ReadFileToMemoryStream(string path)
+        {
+            if (!File.Exists(path))
+                throw new FileNotFoundException($"File not found in {path}");
+
+            var bytes = File.ReadAllBytes(path);
```

Comment:
Calling code around ReadFileToMemoryStream is only catching FileFormatException. What if for example an IOException is thrown here?

Reply:
That is fine. That is something handled by higher level handlers